### PR TITLE
Adding the 404 custom error

### PR DESCRIPTION
**Pull request by efajardo**
This PR is for ticket [3290](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3290). I added the same 404 pages from technology.

### CODE REVIEW

**Comment by brianhlin** [comment]
Again LGTM. @osg-cat ?

File: mkdocs.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ repo_name: null
 repo_url: null
 theme:
   name: windmill
+  custom_dir: osgthedocs
```
Comment (osg-cat) [inline mkdocs.yml]: Same comment here: Choose a clear subdirectory name.

**Comment by efajardo** [comment]
@osg-cat , what about now? Does it look better?

**Comment by osg-cat** [comment]
Yes, thank you

**Comment by efajardo** [comment]
@brianhlin I do not have permission to merge, can you do it?

**Pull request merged**